src/mb_enhanced_cover_art_uploads/providers/musicbrainz.ts

```diff
@@ -0,0 +1,48 @@
+import { assertDefined } from '@lib/util/assert';
+import { safeParseJSON } from '@lib/util/json';
+
+import type { CoverArt } from './base';
+import { ArtworkTypeIDs, CoverArtProvider } from './base';
+
+interface CAAIndex {
+    images: Array<{
+        comment: string;
+        types: string[];
+        id: string | number;  // Used to be string in the past, hasn't been applied retroactively yet, see CAA-129
+        image: string;
+    }>;
+}
+
+export class MusicBrainzProvider extends CoverArtProvider {
+    supportedDomains = ['*.musicbrainz.org'];
+    favicon = 'https://musicbrainz.org/static/images/favicons/favicon-32x32.png';
+    override allowButtons = false;
+    name = 'MusicBrainz';
+    urlRegex = /release\/([a-z0-9-]+)/;
```

Comment:
Good that the regex is not too strict, as you usually want to copy images when you are visiting `/cover-art` of the source release and will likely copy that URL 👍